### ISSUE TITLE
Duplicate Exposure module disabled, increase in exposure via histogram enables and increases exposure in dup exposure

### ISSUE THREAD

**Issue by chhil** [darktable-org/darktable#6545]
darktable-3.3.0+1112~ge0cff08d1-win64.exe

I have exposure and exposure 1 modules. 
Exposure 1 is the duplicate I had created.

Disable exposure 1 module.
Scrolling up in exposure module increases the exposure on Exposure 1 and enables it.

Opencl off.

**Comment by elstoc** [comment]
Working as designed. You can change the behaviour of this in `preferences->miscellaneous->keyboard shortcuts with multiple instances`

**Comment by chhil** [comment]
So, if the preference is not set, the disabled module will get enabled and values changed?

Thats sounds weirdly odd, because there is only one enabled module and that's what should have got changed.

**Comment by elstoc** [comment]
When I introduced the functionality and the preferences I set the defaults in that way in order to best replicate the previous functionality, and that is indeed how it used to work - actually the module that got acted on was fairly random and was either the latest in the pipe or the last created module (or none at all). But shortcuts have never (until I introduced the new preferences) worried about whether or not a module was active.

Either way it's now in your power to choose. Select the "prefer enabled instances" option and it works as you want.
